proto/exchange/matchengine.proto

```diff
@@ -28,7 +28,19 @@ service Matchengine {
       get : "/orders/{market}/{user_id}"
     };
   }
-  rpc OrderCancel(OrderCancelRequest) returns (OrderInfo) {}
+  rpc OrderCancel(OrderCancelRequest) returns (OrderInfo) {
+    option (google.api.http) = {
+      post : "/cancelorder/{market}/{user_id}/{order_id}"
+      body : "*"
+    };
+  }
+  rpc OrderCancelAll(OrderCancelAllRequest) returns (OrderCancelAllResponse) {
+    option (google.api.http) = {
+      post : "/cancelorders/{market}/{user_id}"
+      body : "*"
+    };
+  }
```

Comment:
If I understand correctly, we will need to add authentication in the future. otherwise one can cancel others' orders if knowing user_id?

Reply:
Yes. Currently I use <https://github.com/Fluidex/grpc-dynamic-gateway> to provide a quick API for webpage. It is not the final solution